migrations/versions/0339_service_billing_details.py

```diff
@@ -0,0 +1,39 @@
+"""
+
+Revision ID: 0339_service_billing_details
+Revises: 0338_add_notes_to_service
+Create Date: 2021-01-20 17:55:46.555460
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+revision = '0339_service_billing_details'
+down_revision = '0338_add_notes_to_service'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('services', sa.Column('billing_contact_email_address', sa.Text(), nullable=True))
+    op.add_column('services', sa.Column('billing_contact_name', sa.Text(), nullable=True))
+    op.add_column('services', sa.Column('billing_reference', sa.String(length=255), nullable=True))
+    op.add_column('services', sa.Column('purchase_order_number', sa.String(length=255), nullable=True))
+    op.add_column('services_history', sa.Column('billing_contact_email_address', sa.Text(), nullable=True))
```

Comment:
If we don't need to store the billing history, I'd be curious to know if we can just not add these columns to `services_history`. A quick glance at the code suggests that would work (though I haven't tried it out) - not sure it really matters either way though

Reply:
I would include the history, it would be confusing if the tables were different and it will be useful to have the history for those columns.

Reply:
I can ask Caley tomorrow if having historical records of billing details could be useful. If not, I can try and test what happens when we do not have those columns and update billing details.

Reply:
Sounds like it's good to have the history columns even if we don't look at them from Becca's comment, so I don't think you need to change anything here.

Reply:
Definitely, sorry, Becca's comment didn't load for me yet when I was making my comment.